pb/lmsysmd/model/v1/model.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package lmsysmd.model.v1;
+
+import "buf/validate/validate.proto";
+import "google/type/date.proto";
+
+service ModelService {
+  rpc CreateModel(CreateModelRequest) returns (CreateModelResponse) {
+    option idempotency_level = NO_SIDE_EFFECTS; 
+  }
+  rpc GetModel(GetModelRequest) returns (GetModelResponse) {
+    option idempotency_level = NO_SIDE_EFFECTS; 
+  }
+}
+
+message CreateModelRequest {
+  uint32 model_id = 1 [ (buf.validate.field).required = true ];
+  Model model = 2 [ (buf.validate.field).required = true ];
+}
+
+message CreateModelResponse {
+  uint32 model_id = 1 [ (buf.validate.field).required = true ];
+  Model model = 2 [ (buf.validate.field).required = true ];
```

Comment:
Why is this field needed? Do we need to return the full data? Do we need to return anything at all?